src/components/notifications/DingDing.vue

```diff
@@ -19,8 +19,18 @@
         <select id="mentioning" v-model="$parent.notification.mentioning" class="form-select" required>
             <option value="nobody">{{ $t("Don't mention people") }}</option>
             <option value="everyone">{{ $t("Mention group", { group: "@everyone" }) }}</option>
+            <option value="specify-mobiles">{{ $t("Mention Mobile List") }}</option>
+            <option value="specify-users">{{ $t("Mention User List") }}</option>
         </select>
     </div>
+    <div v-if="$parent.notification.mentioning === 'specify-mobiles'" class="mb-3">
```

Comment:
Could we switch over to `vue-multiselect` for these instead?
I think the UX would be much nicer and less confusing.

What do you think?

Reply:
Yes, it's better for user use, i'll change it